src/panels/lovelace/hui-root.js

```diff
@@ -260,6 +288,7 @@ class HUIRoot extends NavigateMixin(EventsMixin(PolymerElement)) {
   _hassChanged(hass) {
     if (!this.$.view.lastChild) return;
     this.$.view.lastChild.hass = hass;
+    this._updateNotifications();
```

Comment:
This is very expensive. This will now query the websocket backend whenever `hass` changes, which is A LOT. We should cache the websocket response in a different variable and only update that on the `persistent_notifications_updated` event.

Reply:
Oh yeah, that was a bad idea